pkg/pyinst/pyinst-build.mk

```diff
@@ -43,14 +46,12 @@ pyinst-distribution-data:
 
 pyinst-linux-helpers:
 	mkdir -p $(DIST_VERSION)apps/eip/files
-	# TODO compile static
-	cp /usr/sbin/openvpn $(DIST_VERSION)apps/eip/files/leap-openvpn
+	cp ~/leap_thirdparty_build/openvpn $(DIST_VERSION)apps/eip/files/leap-openvpn
```

Comment:
`~/leap_thirdparty_build/`??? Where this is built? Do we have an script to build it?

I think it will make sense to build it inside the bitmask_client folder, I don't like to pollute the home folder.